tariff-management-service/app/src/main/java/com/microservices/tariffmanagement/controller/TariffController.java

```diff
@@ -20,7 +22,17 @@ public TariffController(TariffService tariffService) {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
-    public long createTariff(@Valid @RequestBody TariffDto tariff) {
+    public long createTariff(@Valid @RequestBody CreationTariffDto tariff) {
         return tariffService.createTariff(tariff).getId();
     }
+
+    @GetMapping
+    public List<TariffEntity> getAllTariffs() {
```

Comment:
@andrsuh do we return entities not dtos?

Reply:
We should return entities. Because we are going to place them under "api" module as well as messages and feign client. And then this api module might be imported by other services (API Gateway for example) in order for them not having to duplicate data structures.